integration-tests/tests/resources/samples/print-functions/debug.bal

```diff
@@ -46,5 +46,5 @@ function f2() {
 
 function f3() {
     error e = error("bad sad");
-    log:printDebug("debug log", stackTrace = e.stackTrace().callStack, username = "Alex92", id = 845315);
+    log:printDebug("debug log", stackTrace = e.stackTrace(), username = "Alex92", id = 845315);
```

Comment:
Wouldn’t this be a breaking change?

Reply:
Discussed this with the Choreo team. They are planning to use this feature in the future but they are not using this ATM.

Reply:
This is a breaking change from ballerina-lang. The `error:stackTrace()` function does not return a `CallStack` oblect. Now it returns `error:StackFrame[]`[[1]](https://github.com/ballerina-platform/ballerina-release/blob/beta4-language-changes/release-notes/swan-lake-beta4-release-note.md).

[1] https://github.com/ballerina-platform/ballerina-release/blob/beta4-language-changes/release-notes/swan-lake-beta4-release-note.md

Reply:
Let's add this to.the release note and go ahead.